apps/prairielearn/elements/pl-file-editor/pl-file-editor.js

```diff
@@ -1,7 +1,7 @@
 /* global ace, MathJax, DOMPurify */
 
 window.PLFileEditor = function (uuid, options) {
-  var elementId = '#file-editor-' + uuid;
```

Comment:
One of the only rules I left on was a preference for `const/let` over `var`. The years of `var` are over 😄